gslib/commands/du.py

```diff
@@ -62,7 +62,8 @@
               can use this to make the output machine-readable.
 
   -a          Includes both live and noncurrent object versions. Also prints the
-              generation and metageneration number for each listed object.
+              generation and metageneration number for each listed object. Without
+              specifying this flag, only live object versions are included.
```

Comment:
nit: Could we change the language to "If this flag is not specified, only live object versions are included"? Otherwise looks great.

Reply:
Done. I accidentally pulled from upstream and had to force push to revert it. Please kindly check the final commit and file change.

Reply:
LGTM